mindmaps-engine/src/main/java/io/mindmaps/engine/loader/BlockingLoader.java

```diff
@@ -42,34 +43,35 @@
  */
 public class BlockingLoader extends Loader {
 
-    private ExecutorService executor;
-    private Cache cache;
+    private static ConfigProperties prop = ConfigProperties.getInstance();
+
+    private static int repeatCommits = prop.getPropertyAsInt(ConfigProperties.LOADER_REPEAT_COMMITS);
```

Comment:
It's used elsewhere, most notably in the RESTLoader. This is the config property that we set to indicate the number of retires if a commit fails.